### ISSUE TITLE
Shadowsocks Fails to Install on Debian 8

### ISSUE THREAD

**Issue by antagonismorg** [jlund/streisand#260]
I am attempting install streisand on a DigitalOcean Debian 8 VPS. When installing Shadowsocks, it receives the following error.

> TASK [shadowsocks : Add the Shadowsocks-libev repository] **********************
> An exception occurred during task execution. To see the full traceback, use -vvv. The error was: , E:Some index files failed to download. They have been ignored, or old ones used instead.
> fatal: [streisand.antagonism.org]: FAILED! => {"changed": false, "failed": true, "parsed": false}

The sources.list file reads

> deb http://shadowsocks.org/debian jessie main

According to the Shadowsocks website, it should be

> deb http://shadowsocks.org/debian jessie main

Changing to the latter resolves the issue.

**Comment by df-sh** [comment]
Looks like the same line twice to me... am I missing something?

**Comment by jlund** [comment]
The Shadowsocks website and [shadowsocks-libev repository](https://github.com/shadowsocks/shadowsocks-libev#install-from-repository) don't mention Debian 8 (Jessie), so it's unclear what to do. Here are the repo choices they provide:

```
# Ubuntu 14.04 or above
deb http://shadowsocks.org/ubuntu trusty main

# Debian Wheezy, Ubuntu 12.04 or any distribution with libssl > 1.0.1
deb http://shadowsocks.org/debian wheezy main
```

These choices raise some interesting questions:
* Do you go with `deb http://shadowsocks.org/debian wheezy main` even though you're using Debian **Jessie** (not Wheezy) and even though Debian 8 includes many, many libraries that are _far_ more recent than the ones included in Ubuntu 12.04 and Debian 7?
* Do you go with `deb http://shadowsocks.org/ubuntu trusty main` even though you're not using Ubuntu? Does Debian 8 fall into the "or above" category because it was released after Ubuntu 14.04?
* Why doesn't the Shadowsocks team provide a package for Debian 8 or documentation on what you're supposed to do?

These are great questions! I tried to write the [relevant task](https://github.com/jlund/streisand/blob/master/playbooks/roles/shadowsocks/tasks/main.yml#L12) with an eye towards the future by using variables to populate the repository information. This only works well if the packager also sticks to a sane convention though.

For now, this isn't an issue on the supported distribution. The line is generated properly for Ubuntu 14.04, and Shadowsocks provides an unambiguous official package for Ubuntu 14.04.

Use Ubuntu 14.04 :)
